jenkins/job-configs/kubernetes-jenkins/continuous-docker-validation.yaml

```diff
@@ -19,24 +19,15 @@
 
 # Template for the continuous Docker validation jobs.
 - job-template:
-    name: 'continuous-docker-validation-{os-distro}'
+    name: '{job}'
```

Comment:
nit: looks a little wierd that both jobs are sharing a template but they have nothing in common in names. How about `continuous-docker-validation-{test-suite}`, where `{test-suite}` can be `e2e` or `node-e2e`?